### PR TITLE
Update font stack

### PR DESCRIPTION
**Pull request by shannpersand**
## What Changed?
- Removed main Signika font files and references in main app and switched to Supreme Variable
- Left the pdf generating fonts in for now—not sure what replacement of that would take, but is a secondary concern

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

File: src/styles/_fonts.scss [hunk 1]
```diff
@@ -1,33 +1,26 @@
-/* signika-300 - latin */
 @font-face {
+  font-family: "Supreme-Variable";
+  src: url("./fonts/Supreme/Supreme-Variable.woff2") format("woff2"),
+    url("./fonts/Supreme/Supreme-Variable.woff") format("woff");
+  font-weight: 100 800;
```
Comment (josh-heyer) [inline src/styles/_fonts.scss:5]: For whatever reason, our (b, strong) rules define the bold text used in MDX renders as `bolder` not `bold` - meaning they end up with a weight of 400, which is *very* hard to distinguish from 300 in Supreme-Variable. Observe:

Signika (current):
image

Supreme: 
image

(observed in FF and Chrome on Windows 10)

**Review by josh-heyer — COMMENTED** [review]
Looks fine except for bold text - see comment inline.

File: src/styles/_fonts.scss (hunk 1 above)
Comment (josh-heyer) [reply src/styles/_fonts.scss:5]: (ah, looks like `b, strong { font-weight: bolder; }` is a bootstrap thing)

Comment (shannpersand) [reply src/styles/_fonts.scss:5]: Oo good catch, I'll fix that, thanks!

**Review by josh-heyer — APPROVED** [review]
Looks good!

**Review by frago12 — APPROVED** [review]

**Comment by shannpersand** [comment]
Sorry, need updated reviews (realized I'd missed bumping the weight for all the rendered h tags)

**Review by josh-heyer — DISMISSED** [review]
Good eye!

**Review by frago12 — APPROVED** [review]

**Review by josh-heyer — APPROVED** [review]

**Pull request merged**